Powershell scripts/JIT Scripts/JIT Custom Role/Set-JitLeastPrivilegedRole.ps1

```diff
@@ -1,5 +1,5 @@
 ###############################################################################################
-# This script creates custom Role for Azure Security Center Just-in-time access.              #
+# This script creates custom Role for Azure Defender for Cloud Just-in-time access.              #
```

Comment:
It's Microsoft Defender for Cloud, not Azure Defender for Cloud.

Reply:
Fixed.